src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+
+    private String name = "Square";
```

Comment:
Consider removing the 'name' field and implementing a 'getName()' method that returns the class name using 'getClass().getSimpleName()' or just return the string "Square" directly in 'getName()' if the name is not supposed to change.